_docs/tasks/ingress.md

```diff
@@ -52,48 +59,48 @@ to configure ingress behavior.
    
 1. Determine the ingress URL:
 
-   If your cluster is running in an environment that supports external load balancers,
-   use the ingress' external address:
+   * If your cluster is running in an environment that supports external load balancers,
+     use the ingress' external address:
 
-   ```bash
-   kubectl get ingress -o wide
-   ```
+     ```bash
```

Comment:
Stylistically I think it is easier to read when the command and expected results of executing the command are in the same block.